internal/controller/config.go

```diff
@@ -30,6 +30,10 @@ type Config struct {
 	Tracers                string
 	VerboseMode            bool
 	Version                bool
+	// HostName is the name of the host.
+	HostName string
+	// IPAddress is the IP address of the host.
```

Comment:
nit: as there can be multiple IPs configured, we might want to be more specific.
```suggestion
	// IPAddress is the IP address of the host that sends data to CollAgentAddr.
```